lib/Jsrt/ChakraCore.h

```diff
@@ -733,5 +733,22 @@ JsCopyStringOneByte(
     _Out_opt_ char* buffer,
     _Out_opt_ size_t* written);
 
+/// <summary>
+///     Obtains frequently used properties of a data view.
+/// </summary>
+/// <param name="dataView">The data view instance.</param>
+/// <param name="arrayBuffer">The ArrayBuffer backstore of the view.</param>
+/// <param name="byteOffset">The offset in bytes from the start of arrayBuffer referenced by the array.</param>
+/// <param name="byteLength">The number of bytes in the array.</param>
+/// <returns>
+///     The code <c>JsNoError</c> if the operation succeeded, a failure code otherwise.
+/// </returns>
+CHAKRA_API
+    JsGetDataViewInfo(
```

Comment:
Yeah, I thought the same thing at first, but that method is purely interested in the raw storage.  This method returns the parameters which were passed to the create method and I couldn't see a good way to get them (other than using script).  Since TypedArray and DataView are both views over an ArrayBuffer, I think it makes sense for them to have similar interaction models.